docs/upgrading.md

```diff
@@ -1,48 +1,91 @@
+- [Upgrading to v0.66.0-sumo-0](#upgrading-to-v0660-sumo-0)
+  - [`filelog` receiver: has been removed from sub-parsers](#filelog-receiver-has-been-removed-from-sub-parsers)
+  - [`sending_queue`: require explicit storage set](#sending_queue-require-explicit-storage-set)
+  - [`apache` receiver: turn on feature gates for resource attributes](#apache-receiver-turn-on-feature-gates-for-resource-attributes)
+  - [`elasticsearch` receiver: turn on more datapoints](#elasticsearch-receiver-turn-on-more-datapoints)
+- [Upgrading to v0.57.2-sumo-0](#upgrading-to-v0572-sumo-0)
+  - [`sumologic` exporter: drop support for source headers](#sumologic-exporter-drop-support-for-source-headers)
+- [Upgrading to v0.56.0-sumo-0](#upgrading-to-v0560-sumo-0)
+  - [`sumologic` exporter: drop support for translating attributes](#sumologic-exporter-drop-support-for-translating-attributes)
+  - [`sumologic` exporter: drop support for translating Telegraf metric names](#sumologic-exporter-drop-support-for-translating-telegraf-metric-names)
+- [Upgrading to v0.55.0-sumo-0](#upgrading-to-v0550-sumo-0)
+  - [`filter` processor: drop support for `expr` language](#filter-processor-drop-support-for-expr-language)
+- [Upgrading to v0.52.0-sumo-0](#upgrading-to-v0520-sumo-0)
+  - [`sumologic` exporter: Removed `carbon2` and `graphite` metric formats](#sumologic-exporter-removed-carbon2-and-graphite-metric-formats)
+- [Upgrading to v0.51.0-sumo-0](#upgrading-to-v0510-sumo-0)
+  - [`k8s_tagger` processor: removed `clusterName` metadata extraction option](#k8s_tagger-processor-removed-clustername-metadata-extraction-option)
+    - [How to upgrade](#how-to-upgrade)
+  - [`sumologic` exporter: metadata translation: changed the attribute that is translated to `_sourceName` from `file.path.resolved` to `log.file.path_resolved`](#sumologic-exporter-metadata-translation-changed-the-attribute-that-is-translated-to-_sourcename-from-filepathresolved-to-logfilepath_resolved)
+    - [How to upgrade](#how-to-upgrade-1)
+- [Upgrading to 0.49.0-sumo-0](#upgrading-to-0490-sumo-0)
+  - [Several changes to receivers using opentelemetry-log-collection](#several-changes-to-receivers-using-opentelemetry-log-collection)
+  - [Sumo Logic exporter metadata handling](#sumo-logic-exporter-metadata-handling)
+    - [Removing unnecessary metadata using the resourceprocessor](#removing-unnecessary-metadata-using-the-resourceprocessor)
+    - [Moving record-level attributes used for metadata to the resource level](#moving-record-level-attributes-used-for-metadata-to-the-resource-level)
+
+## Upgrading to v0.92.0-sumo-0
+
+### Exporters: changed retry logic when using persistent queue
+
+In previous versions, when an exporter (e.g. Sumo Logic exporter) was configured to use retries and persistent queue, the data would be retried indefinitely.
+This was because after reaching the retry limit configured in `retry_on_failure.max_elapsed_time`, the data would be put back in the sending queue.
```

Comment:
```suggestion
In previous versions, when an exporter (e.g. Sumo Logic exporter) was configured to use retries and persistent queue, the data would be retried indefinitely as long as the queue wasn't full.
This was because after reaching the retry limit configured in `retry_on_failure.max_elapsed_time`, the data would be put back in the sending queue, if the queue wasn't full.
```